resources/service.go

```diff
@@ -112,7 +113,9 @@ func (s *Services) Start(ctx context.Context, options options.Options) error {
 	g, ctx := errgroup.WithContext(ctx)
 	for name, service := range s.Services {
 		if rutils.IsSelected(options.Services, name) {
-			g.Go(up(service, options, ctx))
+			ctxTimeout, cancel := context.WithTimeout(ctx, time.Second*30)
```

Comment:
create ctxTimeout once and pass it to `errgroup.WithContext(...)` a couple lines about